remote/client/alpha_vantage_remote_client.py

```diff
@@ -11,5 +11,8 @@ def __init__(self):
 
     def get_stock_price_information(self, ticker: str):
         time_series: TimeSeries = TimeSeries(key=settings.ALPHA_VANTAGE_API_KEY)
+        fundamental_Data: FundamentalData = FundamentalData(key=settings.ALPHA_VANTAGE_API_KEY)
         data, _ = time_series.get_quote_endpoint(symbol=ticker)
-        return data
+        dataCompany, _ = fundamental_Data.get_company_overview(symbol=ticker)
+        data['11. currency'] = dataCompany['Currency']
+        return data
```

Comment:
- Name suggestion
- We can create an in-line dictionary to avoid pass in weird keys like this
- Add new line End of File
```suggestion
        company_data_overview, _ = fundamental_Data.get_company_overview(symbol=ticker)
        return {
            "currency": company_data_overview['Currency'],
            "price": timeseries_quote_data['05. price'],
            "ticker": timeseries_quote_data['01. symbol']
        }
```

Reply:
I only change the "fundamental_Data", where is assuming the old name variable.